localbuild/meta.yaml

```diff
@@ -92,6 +92,12 @@ requirements:
     - email_validator
     - keyring
     - dbus-python
+    - flask-login
+    - pysaml2
+    - cherrypy
+    - mako
+    - flask-sqlalchemy
+    - flask-migrate
```

Comment:
please  check if the xmlsec package is the same as the one which is named xmlsec1 in the debian manager.

https://github.com/conda-forge/xmlsec-feedstock

When we can install it by mamba then we don't have to add it by apt to the docker container

Reply:
I think the xmlsec package is not the same as the one which is named xmlsec1 in the Debian manager. The xmlsec package is a Python library that provides bindings for the XML Security Library, while the xmlsec1 package is a command line tool for signature and encryption transformations of XML data.

https://pypi.org/project/xmlsec/
https://packages.debian.org/stable/text/xmlsec1

Reply:
when I try the xmlsec package it installs

```
+ libgcrypt      1.10.1  h166bdaf_0       conda-forge/linux-64     Cached
  + libgpg-error     1.46  h620e276_0       conda-forge/linux-64     Cached
  + libtool         2.4.7  h27087fc_0       conda-forge/linux-64     Cached
  + libxmlsec1     1.2.33  hb6668e7_0       conda-forge/linux-64     Cached
  + xmlsec     
```
the debian xmlsec1 does also install  libxmlsec1  

and when I look 
```
try:
    from saml2.sigver import get_xmlsec_binary
except ImportError:
    get_xmlsec_binary = None

if get_xmlsec_binary:
    XMLSEC_PATH = get_xmlsec_binary(["/opt/local/bin"])
else:
    XMLSEC_PATH = '/usr/bin/xmlsec1'
```

the debian package seems to be a hardcoded fallback which ignores the env. (abuses also None in the condition ...)

```
>>> XMLSEC_PATH = get_xmlsec_binary(["/opt/local/bin"])
>>> XMLSEC_PATH
'/home/mss2023test/mambaforge/envs/mssdev/bin/xmlsec1'
```

Reply:
Because we have saml2 installed we should not require to install a fallback.

Reply:
As it loads this libxmlsec1, we have to look into 
- https://github.com/conda-forge/libxmlsec1-feedstock/blob/main/recipe/meta.yaml
- https://packages.debian.org/bullseye/libxmlsec1